examples/vanilla/index.js

```diff
@@ -34,6 +34,14 @@ function setUrl(url) {
   stream_link.innerText = streamUrl;
 }
 
+function view(){
```

Comment:
Maybe we should implement this as an object with  attach/detach, detach could then be called without any arguments (as it already knows what element it's attached to)